### PR TITLE
plugin: don't load prebuilt "imagenet" weights when inferencing

### PR DESCRIPTION
**Pull request by CandyQiu**
change imagenet weights

### CODE REVIEW

File: packages/plugins/model-define/tensorflow-mobilenet-model-define/src/index.ts [hunk 1]
```diff
@@ -62,30 +68,50 @@ const mobilenetDefine: ModelDefineType = async (data: ImageDataset, args: ModelD
...
   output = GlobalAveragePooling2D()(output);
-  output = Dense(1024, boa.kwargs({
-    activation: 'relu'
-  }))(output);
+  output = Dense(
+    1024,
+    boa.kwargs({
+      activation: 'relu'
+    })
+  )(output);
```
Comment (yorkie) [inline packages/plugins/model-define/tensorflow-mobilenet-model-define/src/index.ts:100]: This change may not be needed.

File: packages/plugins/model-define/tensorflow-mobilenet-model-define/src/index.ts [hunk 2]
```diff
@@ -62,30 +68,50 @@ const mobilenetDefine: ModelDefineType = async (data: ImageDataset, args: ModelD
...
+    boa.kwargs({
+      activation: 'softmax'
+    })
+  )(output);
+  model = Model(
+    boa.kwargs({
+      inputs: model.input,
+      outputs: outputs
+    })
+  );
```
Comment (yorkie) [inline packages/plugins/model-define/tensorflow-mobilenet-model-define/src/index.ts:114]: Ditto

File: packages/plugins/model-define/tensorflow-mobilenet-model-define/src/index.ts [hunk 3]
```diff
@@ -97,25 +123,32 @@ const mobilenetDefine: ModelDefineType = async (data: ImageDataset, args: ModelD
...
+      optimizer: Adam(
+        boa.kwargs({
+          lr: learningRate,
+          decay
+        })
+      ),
+      loss: loss,
+      metrics: metrics
+    })
+  );
```
Comment (yorkie) [inline packages/plugins/model-define/tensorflow-mobilenet-model-define/src/index.ts:137]: Ditto

**Comment by yorkie** [comment]
Because of the bug of `eslint-plugin-react@7.20.4`, may I ask you to rebase your branch on `master` again, we have fixed that at #375 and it gets merged on the `master` branch already.

**Comment by rickycao-qy** [comment]
@CandyQiu Could you format the codes as normal and rebase the master as we discussed before, thx

**Comment by rickycao-qy** [comment]
LGTM

File: packages/plugins/model-define/tensorflow-resnet-model-define/src/index.ts [hunk 4]
```diff
@@ -122,35 +107,28 @@ const resnetModelDefine: ModelDefineType = async (data: ImageDataset, args: Mode
...
       const shape = tf.shape(image).numpy();
-      return this.model.predict(tf.reshape(image, [1].concat(...shape.slice(0, 3)))).toString();
+      return this.model.predict(tf.reshape(image, [ 1 ].concat(...shape.slice(0, 3)))).toString();
     }
   };
   return result;
 };
 
-export default resnetModelDefine;
+export default resnetModelDefine;
```
Comment (WenheLI) [inline packages/plugins/model-define/tensorflow-resnet-model-define/src/index.ts]: We might need an EOL here?

```suggestion
export default resnetModelDefine;

```

**Pull request merged**